docs/UserGuide.md

```diff
@@ -1316,12 +1300,18 @@ Command Words Accepted: `summary` (case-insensitive)
 Important notes:
```

Comment:
May be good to add an important callout indicating that additional parameters will not be handled, like I added for get/clear budget

Reply:
Also, might want to use a callout for the important notes too to highlight that the user should be reading the section below

Reply:
Resolved the first issue. Regarding the second one, if we want to use a callout for all the important notes, then does this apply to all the other commands like delete, get, and find too?

Reply:
If relevant, yes, I believe get and find both have them already